tests/parse/test_parse_case.py

```diff
@@ -198,9 +196,9 @@ def test_parse_case_vcf_files(scout_config, vcf_file):
 
 
 @pytest.mark.parametrize("bam_name", ["alignment_path", "bam_file", "bam_path"])
-def test_parse_case_bams(scout_config, bam_name):
+def test_parse_case_bams(scout_config, bam_name, custom_temp_file):
     # GIVEN a load config with bam_path as key to bam/cram files
-    bam_path = "a bam"
+    bam_path = str(custom_temp_file(".bam"))
```

Comment:
Changed this test to use real paths, because otherwise the validation now fails when a case has bam files associated that are not real files on disk